src/components/HURUmap/Panel/Profile.js

```diff
@@ -126,7 +126,7 @@ const Profile = forwardRef(function Profile(
         <PinAndCompare
           {...props}
           {...pinAndCompare}
-          geography={primaryProfile?.geography?.code}
+          geographyCode={primaryProfile?.geography?.code}
```

Comment:
Good catch. Lets rename `geographyCode` to `currentCode` or `currentGeographyCode` to emphasize why it's needed.